spec/models/import_spec.rb

```diff
@@ -8,11 +8,6 @@
   let(:data) { attachment_fixture('imports.txt') }
 
   describe 'validations' do
-    it 'has a valid parameters' do
-      import = described_class.create(account: account, type: type, data: data)
-      expect(import).to be_valid
-    end
```

Comment:
This actually doesn't use `Fabricate` here so I don't think it quite duplicates the validity check (though I admit this is really close)

Reply:
Yes, agreed that "this well known set of attrs/values creates a valid record" (like in this spec) is different from "our fabricator can create a valid record" so it's not literally a duplication.

In the actual usage here, the fabricator is using the same values as this spec to create the factory instance ... maybe calling it excessive/redundant/etc is more accurate than calling it an exact repetition, but I'd still say its not really worth preserving, given the fabricator spec?